src/main/java/org/fogbowcloud/saps/engine/core/archiver/Archiver.java

```diff
@@ -235,79 +151,44 @@ private void rollBackArchive(SapsImage task) {
                 "updates task [" + taskId + "] to failed state");
 
         addTimestampTaskInCatalog(task, "updates task [" + taskId + "] timestamp");
     }
 
     /**
-     * This function delete pending files from task in Swift.
-     *
-     * @param task task with files information to be deleted
-     * @return boolean representation, success (true) or failure (false) to delete
-     * files
-     * @throws Exception
-     */
-    private boolean deletePendingFilesFromSwift(SapsImage task) throws Exception {
-        String containerName = getContainerName();
-        String taskId = task.getTaskId();
-
-        LOGGER.debug("Deleting files from task [" + taskId + "] in Swift [" + containerName + "]");
-
-        String prefix = properties.getProperty(SapsPropertiesConstants.SWIFT_FOLDER_PREFIX) + File.separator + taskId;
-
-        List<String> fileNames = swiftAPIClient.listFilesWithPrefix(containerName, prefix);
-
-        LOGGER.info("Files List: " + fileNames);
-
-        for (String file : fileNames) {
-            LOGGER.debug("Trying to delete file " + file + " from " + containerName);
-            swiftAPIClient.deleteFile(containerName, file);
-        }
-
-        return true;
-    }
-
-    /**
-     * This function archives finished tasks.
+     * It archives {@code ImageTaskState.FINISHED} {@code SapsImage} in {@code PermanentStorage}.
      */
     private void archiver() {
-        try {
-            List<SapsImage> tasksToArchive = tasksToArchive();
-
-            LOGGER.info("Trying to archive " + tasksToArchive.size() + " finished tasks: " + tasksToArchive);
+        List<SapsImage> tasksToArchive = tasksToArchive();
 
-            for (SapsImage task : tasksToArchive) {
-                LOGGER.info("Try to archive task [" + task.getTaskId() + "]");
-                tryTaskArchive(task);
-                deleteAllTaskFilesFromDisk(task);
-            }
+        LOGGER.info("Trying to archive " + tasksToArchive.size() + " finished tasks: " + tasksToArchive);
 
-        } catch (Exception e) {
-            LOGGER.error("Erro while try archive");
+        for (SapsImage task : tasksToArchive) {
+            LOGGER.info("Try to archive task [" + task.getTaskId() + "]");
+            tryTaskArchive(task);
+            deleteAllTaskFilesFromDisk(task);
         }
     }
 
     /**
-     * This function gets tasks in finished state in Catalog.
+     * It gets {@code SapsImage} list in {@code ImageTaskState.FINISHED} state in {@code Catalog}.
      *
-     * @return tasks in specific state
+     * @return {@code SapsImage} list in {@code ImageTaskState.FINISHED} state
      */
     private List<SapsImage> tasksToArchive() {
         return CatalogUtils.getTasks(catalog, ImageTaskState.FINISHED,
                 "gets tasks with " + ImageTaskState.FINISHED.getValue() + " state");
     }
 
     /**
-     * This function try to archive a task.
+     * It try to archive a {@code SapsImage} in {@code PermanentStorage}.
```

Comment:
delete the method and move its internals to the callee

Reply:
Done.